.cargo/config.toml

```diff
@@ -3,3 +3,7 @@ gen-execution-api       = "run --bin tools -- gen-execution-api"
 scenario                = "run --release --bin tools -- scenario"
 scenario-with-tracing   = "run --release --features tracing --bin tools -- scenario"
 replay-block            = "run --release --bin tools -- replay-block"
+
+# To be able to run unit tests on Linux, support compilation to 'x86_64-unknown-linux-gnu'.
+[target.'cfg(target_os = "linux")']
+rustflags = ["-C", "link-args=-Wl,--warn-unresolved-symbols"]
```

Comment:
Do we have any unit tests that need this now? If not, I'd rather not add it tbh, because if I understand correctly, this will turn linker errors into warnings for all targets which means we might have build errors and not notice it.

Reply:
FYI, it sounds like `napi_build` always does this under the hood because it's linked dynamically.

https://github.com/napi-rs/napi-rs/issues/1005#issuecomment-1011034770
https://github.com/napi-rs/napi-rs/issues/1782#issuecomment-1793980863

I agree that we should avoid setting this for the workspace. I have removed both instances for now, as there are no tests using `edr_napi`